### ISSUE TITLE
persistence blacklist will persist to ceph

### ISSUE THREAD

**Issue by liugs0213** [Alluxio/alluxio#14824]
**Alluxio Version:**
alluxio 2.8.0

**Describe the bug**
We set this parameter alluxio.master.persistence.blacklist=.staging,_temporary,.tmp
However, the files in this directory are still synchronized to ceph
![image](https://user-images.githubusercontent.com/55676558/148874179-44c808cf-cd9c-43a2-ad6d-a90996d8d3fc.png)
![image](https://user-images.githubusercontent.com/55676558/148873559-f8063690-6e24-465c-8dac-32c60bc97dd7.png)
![image](https://user-images.githubusercontent.com/55676558/148873647-83485a66-0789-4980-a2d6-0b5cb67b39f5.png)

**To Reproduce**
Write the same directory using Spark for multiple times.

**Expected behavior**
Files in the _temporary directory are not persist to ceph.

**Comment by yuzhu** [comment]
@liugs0213 persistence blacklist is only used if async persist is used. Could you confirm that async persist is the write mode?

**Comment by liugs0213** [comment]
@yuzhu Yes, we cluster the default settings alluxio.user.file.writetype.default=ASYNC_THROUGH
![image](https://user-images.githubusercontent.com/55676558/149161139-68f12b5c-c7c0-4f63-9241-b5255e710df9.png)

**Comment by lilyzhoupeijie** [comment]
@liugs0213  try to check which spark FileOutputCommitter we are using now after spark update to 3.2.x, and check if this logic has executed ?
![图片](https://user-images.githubusercontent.com/24890599/149745486-02f68309-f704-41d1-add6-32e484dbaac3.png)

try to print some logs before the method shouldPersistPath and print the path

**Comment by liugs0213** [comment]
> @liugs0213 try to check which spark FileOutputCommitter we are using now after spark update to 3.2.x, and check if this logic has executed ? ![图片](https://user-images.githubusercontent.com/24890599/149745486-02f68309-f704-41d1-add6-32e484dbaac3.png)
> 
> try to print some logs before the method shouldPersistPath and print the path

All right, I'll get it tested
